component/kafka/group/option.go

```diff
@@ -26,6 +28,38 @@ func FailureStrategy(fs kafka.FailStrategy) OptionFunc {
 	}
 }
 
+// CheckTopic will attempt to:
+//	1. connect to the broker
```

Comment:
@teivah : locally I don't see it having an indentation that is different from the others...